ansible/deploy-pipeline.yml

```diff
@@ -23,7 +23,7 @@
       login_password: '{{ shovel_postgres_password }}'
       db: metadb # otherwise connection fail
       name: amsrepl
-      password: '{{ amsrepl_hkgmetadb_password }}'
+      password: 'INSECURE_USE_SSL_CERT'
```

Comment:
Probably not, though I recall we were discussing of setting it explicitly to some value to make it clear that it should not be relied upon.

We can perhaps remove it though.

Reply:
Yep, I suggested to have IP-limited usernames with insecure passwords and you suggested to drop the passwords altogether with `PASSWORD NULL` and replace them with certificates, to avoid any sort of secrets being stored in the PG itself.
That's why I was surprised to see insecure password instead of NULL password :-)